tests/test_items.py

```diff
@@ -120,6 +145,49 @@ def test_delete_item__success(self):
         assert resp.status_code == client.NO_CONTENT
         assert not Item.select().exists()
 
+    def test_delete_item__pictures_cascade(self):
+        test_utils.setup_images()
+        item = Item.create(**TEST_ITEM)
+        item2 = Item.create(**TEST_ITEM2)
+        picture = Picture.create(item=item, **TEST_PICTURE)
+        picture2 = Picture.create(item=item, **TEST_PICTURE2)
+        picture3 = Picture.create(item=item2, **TEST_PICTURE3)
+        open("{path}/{picture_id}.{extension}".format(
```

Comment:
Usiamo sempre os.path.join in modo da rendere il codice compatibile con windows (blaaa)